SampleProject/VOKCoreDataManagerTests/BundleTests.m

```diff
@@ -0,0 +1,30 @@
+//
+//  BundleTests.m
+//  SampleProject
+//
+//  Created by Ellen Shapiro (Vokal) on 10/20/15.
+//
+//
+
+#import <XCTest/XCTest.h>
+#import "VOKCoreDataManager.h"
+
+@interface BundleTests : XCTestCase
+
+@end
+
+@implementation BundleTests
+
+- (void)testWorksWithoutSettingCustomBundle
+{
+    XCTAssertNoThrow([[VOKCoreDataManager sharedInstance] setResource:@"VICoreDataModel" database:nil]);
+}
+
+- (void)testWorksWithSettingCustomBundle
+{
+    XCTAssertNoThrow([[VOKCoreDataManager sharedInstance] setResource:@"VICoreDataModel"
```

Comment:
Hm, ok, well, this is obnoxious - in Xcode 7 `XCTAssertThrows` doesn't actually verify exceptions thrown by `NSAssert` statements and `XCTAssertNoThrow` doesn't actually flunk something that throws an exception - it flunks almost coincidentally *because* of the exception.

This is probably related to the fact that a `try/catch` block doesn't actually catch the exception thrown by the `NSAssert`, because I tried that too. :stuck_out_tongue_closed_eyes: 

@vokal-isaac @chillpop any thoughts on how to better test this?